boxtree/translation_classes.py

```diff
@@ -161,54 +165,61 @@ class _KernelInfo(Record):
     pass
 
 
-class RotationClassesInfo(DeviceDataRecord):
-    r"""Interaction lists to help with matrix precomputations for rotation-based
-    translations ("point and shoot").
+class TranslationClassesInfo(DeviceDataRecord):
+    r"""Interaction lists to help with for translations that benefit from
+    precomputing distance related values
 
-    .. attribute:: nfrom_sep_siblings_rotation_classes
+    .. attribute:: nfrom_sep_siblings_translation_classes
 
-       The number of distinct rotation classes.
+       The number of distinct translation classes.
 
-    .. attribute:: from_sep_siblings_rotation_classes
+    .. attribute:: from_sep_siblings_translation_classes
 
         ``int32 [*]``
 
         A list, corresponding to *from_sep_siblings_lists* of *trav*, of
```

Comment:
```suggestion
        A list, corresponding to *from_sep_siblings_lists* of :attr:`traversal`, of
```
(i.e. add that as an attribute, to clarify what it refers to)

Reply:
`traversal` is not an attribute of the `TranslationClassesInfo` object though. It's an argument to the builder.

Reply:
What I meant to add the traversal as an attribute even though it's not functionally required, just so that the documentation can refer to it.

Reply:
Ah. I've added it.